Signal/src/ViewControllers/Utils/MessageRecipientStatusUtils.swift

```diff
@@ -159,4 +159,24 @@ public class MessageRecipientStatusUtils: NSObject {
                                                               referenceView: referenceView)
         return status
     }
+
+    @objc
+    public class func description(forMessageReceiptStatus value: MessageReceiptStatus) -> String {
```

Comment:
Is this just for logging?

Reply:
Yup.  I try to keep around formatting methods for any enums that we might want to log, because... I often find myself (temporarily) logging them.